### ISSUE TITLE
Create train and test set from Trump tweet threads so that I can train a model fit to Trumps data

### ISSUE THREAD

**Issue by scramblingbalam** [scramblingbalam/Alta_Real#45]
Follow a similar process to that laid out in "Crowdsourcing the Annotation of Rumourous
Conversations in Social Media" to create an extended dataset of Trump tweets.
This data set should consist of roughly 1000 labeled tweets, and should include multiple threads.
Since the the biggest deficiency int he the RumorEval training set is the lack of denying tweets I should pick threads that have will likely have more denying tweets.
The same is true for  supporting tweets so that would be a second priority

**Comment by scramblingbalam** [comment]
performance with current train set of 3 events covering 1 thread each:
Note: event names are given by the beginning of the string of the root: 

event: Big_win_in_the_House
id: 860477328882905089 
edge_list.length = 2377

event: Eventually,_but_at_a
id: 856172056932700164 
edge_list.length = 3

event: #WeeklyAddress🇺🇸_htt
id: 860580764944969728 
edge_list.length = 2339

ottawashooting
twit_doc2vec_300: 0.679536679537
ebola-essien
twit_doc2vec_300: 0.5
Big_win_in_the_House
twit_doc2vec_300: 0.36795626577
#WeeklyAddress🇺🇸_htt
twit_doc2vec_300: 0.340598290598
charliehebdo
twit_doc2vec_300: 0.729225023343
putinmissing
twit_doc2vec_300: 0.629032258065
germanwings-crash
twit_doc2vec_300: 0.710714285714
sydneysiege
twit_doc2vec_300: 0.695573622403
prince-toronto
twit_doc2vec_300: 0.747572815534
Eventually,_but_at_a
twit_doc2vec_300: 0.5
ferguson
twit_doc2vec_300: 0.656826568266
(u'twit_doc2vec_300', 'feature-list')
                               precision       recall          f1              n              
supporting                     0.523           0.282           0.366           1675           
denying                        0.310           0.004           0.007           2446           
appeal-for-more-information    0.355           0.503           0.416           467            
comment                        0.534           0.877           0.664           4652           

Macroevaluation                0.430           0.416           0.363           9240           
Microevaluation                0.519           0.519           0.519           9240
